doc/api/packages.md

```diff
@@ -79,10 +80,10 @@ import './node_modules/commonjs-package/index.js';
 ```
 
 Files ending with `.mjs` are always loaded as [ES modules][] regardless of
-package scope.
+package boundary.
 
 Files ending with `.cjs` are always loaded as [CommonJS][] regardless of package
```

Comment:
```suggestion
Files ending with `.cjs` are always loaded as [CommonJS][] regardless of the
nearest parent `package.json`.
```

(Suggest tool won’t let me include the following line, make sure to delete the orphaned “boundary”.)